cluster/gce/gci/configure-helper.sh

```diff
@@ -1423,6 +1423,9 @@ function start-kube-addons {
   if [[ "${ENABLE_DEFAULT_STORAGE_CLASS:-}" == "true" ]]; then
     setup-addon-manifests "addons" "storage-class/gce"
   fi
+  if [[ "${ENABLE_METADATA_PROXY:-}" == "simple" ]]; then
```

Comment:
Why only in GCI? You should add it at least to debian, as some of our tests run on it.